AntistasiOfficial.Altis/Save/fn_loadGame.sqf

```diff
@@ -66,14 +66,10 @@ unlockedRifles = unlockedweapons - gear_sidearms - gear_missileLaunchers - gear_
 //["ws_grid"] call fn_loadData;
 //===========================================================================
 
-
 {
 	[(_x select 0), (_x select 1)] remoteExec ["createSupplyBox", call AS_fnc_getNextWorker];
 } forEach supplySaveArray;
 
-countSupplyCrates = count supplySaveArray;
-publicVariable "countSupplyCrates";
```

Comment:
If this is deleted, Line 155 (I don´t know how to properly intergrate it in the text) of resourcescheck.sqf is obsolete too.
On quick save and load again and again this will cause an unlimited amount of crates to spawn in!